source/documentation/getting_started/get_started.md

```diff
@@ -6,9 +6,11 @@ Your department, agency or team must have a GOV.UK PaaS account. This account is
 
 Once your department, agency or team has an org account, you will need an individual account. Ask your [org manager](orgs_spaces_users.html#org-manager) to authorise the creation of your individual account.
 
+The GOV.UK PaaS is hosted in two locations, London and Ireland. The GOV.UK PaaS team creates all new orgs on the London-based GOV.UK PaaS by default. If you already have an account on the Ireland-based GOV.UK PaaS and request a new org, we will create this new org and account on the London-based GOV.UK PaaS, and you will have two separate accounts.
```

Comment:
The cloud-y world has settled on "region" as a term which expresses both geographic distance between "clouds", and their operational independence.  Would that perhaps be a more suitable/aligned term to use than "location", throughout?